Backend/Models/Users/BannedIp.cs

```diff
@@ -25,9 +28,22 @@ public void SaveToDb()
             context.SaveChanges();
         }
 
-        public bool AlreadyExists()
+        public void RemoveFromDb()
         {
-            List<BannedIp> ipsFromDb = context.BannedIps.Where(m => m.IpAddress == IpAddress).ToList();
+            context.Remove(this);
+            context.SaveChanges();
+        }
+
+        public static BannedIp? GetBannedIpFromDb(string bannedIp)
+        {
+            List<BannedIp> bannedIpsFromDb = context.BannedIps.Where(m => m.IpAddress == bannedIp).ToList();
+            if (bannedIpsFromDb.Count == 0) return null;
```

Comment:
wrap the return in brackets please, we're not using single line ifs

Reply:
Not even my code smh